docs/user-manual.md

```diff
@@ -1,229 +1,263 @@
       - if: failure()
         run: job-failure
 ```
 
-How to specify the channel for Slack notifications
---------------------------------------------------
+## Slack notifications
+
+By default, meows sends the job result to the slack channel specified by the `slack-app-secret` secret.
+However, you can change the slack channel in several methods.
+Each method accepts a channel name in the `#<channel_name>` format. (e.g. `#general`, `#test1`)
 
-The following methods exist for specifying the channel for Slack notifications.
-The priority order of the specification method is 4>3>2>1.
-Any method accepts a channel name in the format of `#<channel_name>`. (e.g. `#general`, `#test1`)
+1. Call `meows slackagent set-channel "#channel"` command in a workflow.
+2. `MEOWS_SLACK_CHANNEL` environment variable in a job.
+3. `.spec.slackAgent.channel` field in a `RunnerPool` resource. See [SlackAgentConfig](crd-runner-pool.md#SlackAgentConfig).
+4. `SLACK_CHANNEL` value in the `slack-app-secret` secret. See [Deploying Slack Agent](#deploying-slack-agent).
 
-1. `SLACK_CHANNEL` value in the `slack-app-secret` secret. See [How to deploy Slack agent](#how-to-deploy-slack-agent).
-2. `.spec.slackAgent.channel` field in a `RunnerPool` resource. See [SlackAgentConfig](crd-runner-pool.md#SlackAgentConfig).
-3. `MEOWS_SLACK_CHANNEL` environment variable in a workflow.
-4. Call `meows slackagent set-channel "#channel"` command in a workflow.
+When you specify some channels using some methods, the smaller number is a priority.
+If you don't set any channel in any method, meows does not send a message.
 
 For example, you can specify the channel in a workflow as follows.
 
 ```yaml
-name: slack-channel-specified
+name: slack notification example
 on: push
 
 jobs:
-  build:
-    name: job-name
+  example:
+    name: example
     env:
       # Basically, a job result will be reported to the "#test1" channel.
       MEOWS_SLACK_CHANNEL: "#test1"
     steps:
       - run: job-started
+
       # Only when a job fails, the result will be reported to the "#test2" channel.
-      - run: meows slackagent set-channel "#test2"
-        if: failure()
-      - run: job-success
+      - if: failure()
+        run: meows slackagent set-channel "#test2"
+
+      - run: ...
+      - run: ...
+      - run: ...
+
+      - if: success()
+        run: job-success
+      - if: cancelled()
+        run: job-cancelled
+      - if: failure()
+        run: job-failure
 ```
 
-How to extend GitHub Actions jobs
----------------------------------
+## Runner pods extension
+
+When a job fails, meows sends the following Slack message.
 
 ![failure message](./images/slack_failure.png)
 
-Choose the time in UTC you want to extend the `Pod` by and click `Extend`.
-This button can be clicked multiple times if the `Pod` still exists.
+If you want to extend the pod, choose the time in UTC and click the `Extend` button.
+You can click the button multiple times if the pod still exists.
```

Comment:
I think this is a bit confusing.
How about `As long as the pod exists, you can extend it as many times as you want.`